app_dart/lib/src/request_handlers/reserve_task.dart

```diff
@@ -194,7 +194,7 @@ class TaskProvider {
   final DatastoreService datastore;
 
   Future<FullTask> findNextTask(Agent agent) async {
-    await for (Commit commit in datastore.queryRecentCommits()) {
+    await for (Commit commit in datastore.queryRecentCommitsNoBranch()) {
```

Comment:
Did you forget uploading the file with this method? or was already there?

Reply:
It was already there: https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/service/datastore.dart#L61

Reply:
Thank!